### ISSUE TITLE
safe target fails when the existing target doesn't exist

### ISSUE THREAD

**Issue by teancom** [starkandwayne/safe#160]
If you have targeted a vault that no longer exists in your `saferc`, further `safe target <x>`s will fail, even if the _new_ target does exist.

To replicate, make this your `~/.saferc`:

```
version: 1
current: aws-vault-1
vaults:
  aws-vault-2:
    url: https://1.2.3.4:8200
    token: ""
    skip_verify: false
```

Then run `safe target aws-vault-2`.

It fails with:

```
!!! Current target 'aws-vault-1' not found in ~/.saferc
```

Expected result:

`safe target` will succeed, if the new target exists, even if the old target doesn't any longer.

Work-around:

Not edit my `~/.saferc` directly which would have avoided this whole mess.

**Comment by jhunt** [comment]
Fix is in, on PR #179

**Comment by thomasmmitchell** [comment]
Should be in 1.1.0
